manuscript.md

```diff
@@ -33,10 +33,39 @@ Yet another of the issues pertains to
 
 ### Network structure
 
-<!-- Most important right now -->
+Research outputs are typically papers, which report on at least one research cycle after it has occurred. The communicative design of papers embeds hindsight and its biases in the reporting of results. Moreover, this design eliminates the verification of the chronology within a paper. On the other hand, the unit of a paper is encompasses so much that citations to other papers can indicate a tangent or a crucial link. Additionally, the paper is a bottleneck for what is communicated: It cannot properly deal with code, data, materials, etc.
+
+When stages of research are communicated separately and as they occur, it changes the communicative design to eliminate hindsight and allows more types of outputs to be communicated. For example, a theory can be communicated first and hypotheses communicated afterwards, as a direct descendant of the theory. Subsequently, a design can be linked as a direct descendant of the hypotheses, materials as a direct descendant of the design, and so on. This would allow for the incorporation of materials, data, and analysis code (amongst others).
+
+Figure 1 shows a simple example of how these different research stages (i.e., nodes) would directly connect to each other. The connection between these nodes only shows the direct descendance and could still include citations to other pieces of information. For example, a discussion section could be a direct descendant of a results section and could still include citations to other relevant findings.
+
+[![Directed Acyclic Graph](https://upload.wikimedia.org/wikipedia/commons/c/c6/Topological_Ordering.svg)](https://en.wikipedia.org/wiki/Directed_acyclic_graph) 
+
+**Figure 1. A Directed Acyclic Graph (DAG) of connected research stages. The ordering is chronological (top-bottom) and therefore nodes that are situated below one another cannot refer upwards.**
+
+Given that these piecemeal units are communicated as they occur, chronology is embedded in the communication process with many added benefits. For example, preregistration of hypotheses tries to ensure that predictions precede observations, which would be embedded with such piecemeal communication. Moreover, if research outputs are communicated as they are produced, selective reporting (i.e., publication bias) is reduced by having already communicated the data before results are generated. 
```

Comment:
Add something like: "Furthermore, communicating in piecemeal units shortens the time with which research output is available to the public. This increases the speed of communication and allows for faster diffusion of knowledge."